src/js/components/Drop/DropContainer.js

```diff
@@ -166,32 +166,27 @@ const DropContainer = forwardRef(
           let bottom;
           let maxHeight = containerRect.height;
           if (align.top) {
-            if (align.top === 'top') {
-              ({ top } = targetRect);
-            } else {
-              top = targetRect.bottom;
-            }
-
-            // Calculate visible area underneath the control w.r.t window height
-            const percentVisibleAreaBelow =
-              100 - (targetRect.bottom / windowHeight) * 100;
+            const alignToTop = align.top === 'top';
+            const switchTarget = !alignToTop
+              ? targetRect.top
+              : targetRect.bottom;
+            top = alignToTop ? targetRect.top : targetRect.bottom;
```

Comment:
My head is a bit confused trying to keep track of all of this. I'm wondering if there's a way to make it a bit clearer, perhaps by renaming some variables, or perhaps by restructuring the code. For instance, would it be clearer if we just checked `if (align.top === 'top') {` and then put everything for that case under it and then `} else {` for the other version, rather than trying to merge them together?